google/cloud/iam/README.md

```diff
@@ -1,10 +1,101 @@
 # Google Cloud IAM C++ Client Library
 
-:construction:
```

Comment:
Does this mean the IAM library is GA?  Should we say so at the top-level README?